sqlmesh/dbt/adapter.py

```diff
@@ -325,7 +341,11 @@ def _map_table_name(
 
     def _relation_to_table(self, relation: BaseRelation) -> exp.Table:
         assert relation.identifier is not None
-        return exp.table_(relation.identifier, db=relation.schema, catalog=relation.database)
+        return exp.table_(
+            relation.identifier,
+            db=relation.schema,
+            catalog=self._get_database(relation.database) if relation.schema else relation.database,
```

Comment:
[Nit] So if the schema is missing we'll just set the database value even if it's available? Why not just None?

Reply:
Going to review some of this stuff with Chris on Monday. I think some of this can be removed since I think he said `database` is always defined therefore we never fallback to `default_catalog`. Will confirm.